### PR TITLE
DOC: Clarify that GetTransformBetweenNodes is always in PostMultiply mode

### PR DESCRIPTION
**Pull request by mauigna06**

### CODE REVIEW

**Review by lassoan — COMMENTED** [review]
Thank you, this may help users.

We don't make a commitment that the transform will be pre or post multiply, so please update the text to something like that "the method may change the PreMulitiply/PostMulitply flags in the transform". Thank you!

File: Libs/MRML/Core/vtkMRMLTransformNode.h [hunk 1]
```diff
@@ -120,29 +120,35 @@ class VTK_MRML_EXPORT vtkMRMLTransformNode : public vtkMRMLDisplayableNode
...
+  /// Resulting transformToWorld multiply order will be overwritten to PostMultiply
   /// \sa GetTransformBetweenNodes
   void GetTransformToWorld(vtkGeneralTransform* transformToWorld);
 
   ///
   /// Get concatenated transforms from world.
+  /// Resulting transformFromWorld multiply order will be overwritten to PostMultiply
   /// \sa GetTransformBetweenNodes
   void GetTransformFromWorld(vtkGeneralTransform* transformToWorld);
+  void GetTransformFromWorld(vtkGeneralTransform* transformFromWorld);
```
Comment (lassoan) [inline Libs/MRML/Core/vtkMRMLTransformNode.h:132]: Is this change intentional?

Comment (mauigna06) [reply Libs/MRML/Core/vtkMRMLTransformNode.h:132]: yes

Comment (lassoan) [reply Libs/MRML/Core/vtkMRMLTransformNode.h:132]: This line causes a build error. You need to remove the old line.

Comment (mauigna06) [reply Libs/MRML/Core/vtkMRMLTransformNode.h:132]: Oh, I redid the commit so many times that that slipped through

**Review by lassoan — APPROVED** [review]
